core/client/pom.xml

```diff
@@ -256,7 +221,7 @@
               <relocations>
                 <relocation>
                   <pattern>org.apache.thrift</pattern>
-                  <shadedPattern>shaded.alluxio.org.apache.thrift</shadedPattern>
+                  <shadedPattern>${project.artifactId}.org.apache.thrift</shadedPattern>
```

Comment:
how about still inlining the `${project.artifactId}` but converting hypen to underscore?